### PR TITLE
read version from package.json

### PR DESCRIPTION
**Pull request by tessalt**
rather than calling `exec` to get git tag, get version number for cdn scripts from package.json. I think this is a better approach but I'm also slightly suspicious about what my original motives were for not doing this the first time around. @minasmart @yomexzo @richgilbank

### CODE REVIEW

File: scripts/deploy [hunk 1]
```diff
@@ -31,9 +31,15 @@ var Uploader = function () {
   this.secrets = new Secrets();
   this.secrets.load();
   this.s3 = new AWS.S3();
+  this.tag = this.getPkgVersion();
   this.distAssets = fs.readdirSync(DIST_DIR);
 };
 
+Uploader.prototype.getPkgVersion = function () {
+  var pkg = fs.readFileSync('./package.json')
```
Comment (minasmart) [inline scripts/deploy]: You can actually do `require('./package.json')` I think. Which will remove the need to manually parse json on the next line.

File: scripts/deploy [hunk 2]
```diff
@@ -53,24 +59,12 @@ Uploader.prototype.uploadFile = function(localPath, s3Path) {
...
-  }.bind(this));
-};
-
-Uploader.prototype.getCurrentTag = function (cb) {
-  exec('git describe --always --tag --abbrev=0', (err, stdout) => {
-    if (err) {
-      console.error(err);
-      cb();
+  this.distAssets.forEach(function (file) {
+    if (file.indexOf(FILENAME_FILTER) > -1) {
```
Comment (minasmart) [inline scripts/deploy]: use `file.contains(FILENAME_FILTER)`

File: scripts/deploy (hunk 1 above)
Comment (tessalt) [inline scripts/deploy]: nice

**Comment by richgilbank** [comment]
:+1:

**Comment by minasmart** [comment]
:shipit:

**Pull request merged**